doc/internals/Core.md

```diff
@@ -44,7 +41,14 @@ through a Decoder, producing a data tree. Then, it uses a Navigator to guide a V
 constructs the final object graph.
 
 To construct the object graph, the Visitor is told which type is associated with the root of the data tree. It is
-then up to the Visitor to figure out which type to associate with each of the other nodes.
+then up to the Visitor to figure out which type to associate with each of the underlying nodes.
+
+SerializerFacade
+----------------
+
+There is a facade that ties everything together. It has a `serialize()` and `deserialize()` method that take a format
+as a parameter. It uses a set of Serializer and Deserializer factories to instantiate de appropriate object, which does
+the real work.
```

Comment:
de -> the, add "The object is discarded once it has done its work. This is a good thing. Serializers and Deserializers use Visitors, which are unsafe for reuse, since they are stateful."